docs/getting_started/install_ez_platform.md

```diff
@@ -1,12 +1,200 @@
 # Install eZ Platform
 
-## General installation guide for all eZ distributions
+## Prepare work environment
 
-This installation guide can be followed with any eZ distribution based on the Symfony framework. Here's a list of available distributions from eZ:
+To install eZ Platform you need a stack with your operating system, MySQL and PHP.
 
-|Type | Archive | License | GIT/Composer|
-|------|------|------|------|
-|**eZ Platform**</br> *"clean distribution"* | [ezplatform.com](https://ezplatform.com/#download-option) | GPLv2 | [ezsystems/ezplatform](https://github.com/ezsystems/ezplatform) ([INSTALL.md](https://github.com/ezsystems/ezplatform/blob/master/README.md#installation))|
-|eZ Platform Demo</br> *not suited for production use* | Available via Git / Composer | GPLv2 | [ezsystems/ezplatform-demo](https://github.com/ezsystems/ezplatform-demo)|
-|**eZ Platform Enterprise Edition**</br> *clean distribution* | [support.ez.no/Downloads](https://support.ez.no/Downloads) | BUL (requires eZ Enterprise subscribtion) | [ezsystems/ezplatform-ee](https://github.com/ezsystems/ezplatform-ee) ([INSTALL.md](https://github.com/ezsystems/ezplatform-ee/blob/master/README.md#installation))|
-|eZ Platform Enterprise Edition Demo</br> *not suited for production use* | [support.ez.no/Downloads](https://support.ez.no/Downloads) | BUL (requires eZ Enterprise subscribtion) | [ezsystems/ezplatform-ee-demo](https://github.com/ezsystems/ezplatform-ee-demo)|
+[For production](#prepare-installation-for-production) you also need Apache as the HTTP server.
+[For development](#use-phps-built-in-server), you can use the built-in PHP server instead.
+
+You also need `git` for version control.
+
+Before getting started, make sure you review the [requirements](requirements.md) page to see the systems we support and use for testing.
+
+!!! note
+
+    Installation for production is only supported on Linux.
+
+    To install eZ Platform for development on Mac OS or Windows,
+    see [a cookbook recipe on installing on those systems](../cookbook/installing-on-mac-os-and-windows.md).
+
+## Get Composer
+
+Install Composer, the PHP command line dependency manager. Use your system package manager such as `apt` on Ubuntu.
```

Comment:
This command will not work for every Linux version only for Ubuntu / Mint / Lubuntu / Xubuntu.
For Fedora / RedHat we should use `rpm`
Or add a note that this command may vary depending on Linux version, for Ubuntu is this...